packages/autocomplete-js/src/__tests__/autocomplete.test.ts

```diff
@@ -156,6 +155,162 @@ describe('autocomplete-js', () => {
     `);
   });
 
+  test('renders empty template on empty results', async () => {
+    const container = document.createElement('div');
+    const panelContainer = document.createElement('div');
+
+    document.body.appendChild(panelContainer);
+    autocomplete<{ label: string }>({
+      id: 'autocomplete',
+      container,
+      panelContainer,
+      debug: true,
+      openOnFocus: true,
+      shouldPanelShow: () => true,
```

Comment:
I had to define this on the tests to keep the `panel` opened. I've tried focusing, waiting etc. but I can't figure out how to do it without forcing it to open, any ideas?

Reply:
In the current implementation, `hasEmptySourceTemplateRef` is initialized before rendering an AC instance. As the default value is `false`, `shouldPanelShow` returns `false` at init even when a template is given.

The issue is not noticeable on browser because it's re-computed on keypress event, however it looks like it's not triggered in the tests, even after firing events.

Any help/ideas is appreciated!